site/owid.scss

```diff
@@ -501,42 +500,22 @@ div.owid-presentations li:nth-of-type(1) a:hover {
 
     /* Subsections e.g. I.1 Historical perspective */
     > h3 {
-        @include h3-style;
-        margin-top: 5rem;
-        margin-bottom: 2.5rem;
-        padding-top: 2.5rem;
-        border-top: 1px solid $blue-20;
+        @include h2-bold;
```

Comment:
![image](https://user-images.githubusercontent.com/11844404/215841523-663a397f-c88b-4093-9f7f-c77d3d64c8a6.png)

I think these h3 headings are currently used in a way that _needs_ the top border. Otherwise, the headings need to be changed to be in the left column only.

I think standardising the headings is good in general, but it should probably have a corresponding data migration to go with it.